rpcs3/rpcs3qt/vfs_dialog_path_widget.cpp

```diff
@@ -55,6 +48,9 @@ vfs_dialog_path_widget::vfs_dialog_path_widget(const QString& name, const QStrin
 
 	setLayout(vbox);
 
+	// Disable focus at startup. It will be eventually enabled later by an explicit row click from the user
+	m_dir_list->setFocusPolicy(Qt::NoFocus);
```

Comment:
This breaks keyboard selection

Reply:
Reviewed the logic making use of stylesheet to apply highlight on selected row. Please re-read the description of the PR and the new screenshots